src/main/java/br/com/studiesMaterials/handlers/ArticleHandler.java

```diff
@@ -0,0 +1,101 @@
+import br.com.studiesMaterials.web.api.schemas.ArticleSchema;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyRequestEvent;
+import com.amazonaws.services.lambda.runtime.events.APIGatewayProxyResponseEvent;
+import com.google.gson.Gson;
+import java.sql.*;
+
+public class ArticleHandler implements ArticleDao {
+    @Override
+    public APIGatewayProxyResponseEvent create(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+
+            String studentId =  input.getPathParameters().get("student_id");
+
+            ArticleSchema data = convertBody(input.getBody());
+
+            String sql = String.format(
+                    "INSERT INTO public.article (student_id, subject, link) " +
+                            "VALUES ('%s', '%s', '%s')", studentId, data.subject, data.link
+            );
+
+            statement.executeUpdate(sql);
+
+            responseEvent.setStatusCode(201);
+
+            return responseEvent;
+        } catch (SQLException error) {
+            error.printStackTrace();
+            responseEvent.setStatusCode(500);
+
+            return responseEvent;
+        }
+    }
+
+    @Override
+    public APIGatewayProxyResponseEvent update(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+
+            String articleId = input.getPathParameters().get("article_id");
+
+            ArticleSchema data = convertBody(input.getBody());
+
+            String sql = String.format(
+                    "UPDATE public.article SET subject = '%s', link = '%s' " +
+                            "WHERE id = '%s'", data.subject, data.link, articleId
+            );
+
+            statement.executeUpdate(sql);
+            conn.close();
+
+            responseEvent.setStatusCode(201);
+
+            return responseEvent;
+        } catch (SQLException error) {
+            error.printStackTrace();
+            responseEvent.setStatusCode(500);
+
+            return responseEvent;
+        }
+    }
+
+    @Override
+    public APIGatewayProxyResponseEvent delete(APIGatewayProxyRequestEvent input) {
+        APIGatewayProxyResponseEvent responseEvent = new APIGatewayProxyResponseEvent();
+        try(Connection conn = DataBase.connection()) {
+            assert conn != null;
+            Statement statement = conn.createStatement();
+            String articleId = input.getPathParameters().get("article_id");
+
+            String sql = String.format("DELETE FROM public.article WHERE id = '%s'", articleId);
+
+            statement.executeUpdate(sql);
+
```

Comment:
Fechar a conexão depois de executar o comando sql.
```suggestion
   conn.close();
```